### DISCUSSION TITLE
[Question]: About the display of plugins

### DISCUSSION THREAD

**Discussion by k-toki** [danny-avila/LibreChat#3756] [Q&A]
### What is your question?

Can I show or hide individual plugins in the plugin store?

### More Details

No response

### What is the main subject of your question?

Documentation, UI

### Screenshots

No response

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Yes, you need to make use of the `librechat.yaml` file: 

https://www.librechat.ai/docs/configuration/librechat_yaml/object_structure/config#filteredtools
